### ISSUE TITLE
Ansible Indention Error

### ISSUE THREAD

**Issue by schmitch** [openshift/openshift-ansible#401]
Hello, currently I get on Tag v3.0.0 and on master the following ansible error:

```
failed: [192.168.21.131] => {"failed": true, "parsed": false}
BECOME-SUCCESS-asqdesikhfgbvuevrgyzhbrbqgppwxyt
  File "/home/vagrant/.ansible/tmp/ansible-tmp-1438173912.0-56297342091673/openshift_facts", line 4918
    if os.path.exists('/proc/self/status'):
                                          ^
IndentationError: unindent does not match any outer indentation level
OpenSSH_6.2p2, OSSLShim 0.9.8r 8 Dec 2011
debug1: Reading configuration data /Users/schmitch/.ssh/config
debug1: Reading configuration data /etc/ssh_config
debug1: /etc/ssh_config line 20: Applying options for *
debug1: /etc/ssh_config line 102: Applying options for *
debug1: auto-mux: Trying existing master
debug1: mux_client_request_session: master session id: 2
Shared connection to 192.168.21.131 closed.
```

Currently I'm using standard vagrant boxes

**Comment by schmitch** [comment]
Looks like I've hit the Ansible 1.9 Bug

**Comment by sudhirpandey** [comment]
@schmitch  did you get workaround this error. I am getting the same error.

**Comment by detiber** [comment]
@sudhirpandey what version of ansible are you running? We had an issue using ansible 1.9.0, but it should result in an failed task now as opposed to the error seen above.

**Comment by sudhirpandey** [comment]
@detiber exactly , it turned out to be i was using ansible 1.9.0.1 and is fixed now when upgraded to 1.9.4
